test/Unit/AESCryptoTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace NCrypt\Client;
+
+use NCrypt\Client\Library\Crypto\AESCrypto;
+use NCrypt\Client\Model\SecureNote;
+use PHPUnit\Framework\TestCase;
+
+class AESCryptoTest extends TestCase
+{
+
+    public function testReturnTypeValidatorForSecureNoteRequest()
+    {
+        $crypto = new AESCrypto();
+        $secureNoteResult = $crypto->secureNoteRequest("this is a test note");
+        $this->assertInstanceOf(SecureNote::class,$secureNoteResult);
```

Comment:
Kindly change it to `assertNotNull` this would suffice.

Reply:
it's done